.github/workflows/build_with_conda.yml

```
@@ -16,13 +21,18 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+
       - name: Install Conda environment from environment_unix.yml
         uses: mamba-org/provision-with-micromamba@main
-
         with:
           environment-file: environment_unix.yml
           environment-name: arcticdb
-
+          
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.8.0
+        with:
+          mongodb-port: 27017
```

According [the `README` of this action](https://github.com/supercharge/mongodb-github-action#introduction), the version of MongoDB must be specified.
```suggestion
          mongodb-version: 5.0
          mongodb-port: 27017
```